src/coreclr/tools/ILVerify/Program.cs

```diff
@@ -34,7 +34,7 @@ public Program(ILVerifyRootCommand command)
             _command = command;
 
             _inputFilePaths = Get(command.InputFilePath);
-            _referenceFilePaths = Get(command.Reference);
+            _referenceFilePaths = Get(command.Reference) ?? new Dictionary<string, string>();
```

Comment:
Ops, sorry about this. It was a breaking change in a later version of command-line-api. The recommended way in that breaking change was how we handled default case in crossgen2 and ilc: https://github.com/dotnet/runtime/commit/af6e7fe132ada839b61bfecdc355354e876323f8

Reply:
Could you please submit a PR with the fix you are proposing? (I am going to close this one.)